SPECS/emacs/emacs.spec

```
@@ -0,0 +1,86 @@
+Summary:        GNU Emacs text editor
+Name:           emacs
+Version:        28.1
+Release:        2%{?dist}
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+License:        GPLv3+ and CC0-1.0
+URL:            http://www.gnu.org/software/emacs/
+Group:          Applications/Editors
+Source0:        https://ftp.gnu.org/gnu/emacs/emacs-%{version}.tar.xz 
```

Nit: Could use macros for the package name

```suggestion
Source0:        https://ftp.gnu.org/gnu/%{name}/%{name}-%{version}.tar.xz 
```

---

updated